extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/GraalVMFeatures.java

```diff
@@ -55,4 +57,12 @@ ReflectiveClassBuildItem registerNamingStrategiesForReflections() {
                 .build();
     }
 
+    @BuildStep(onlyIf = { HibernateSpatialAvailable.class, GeolatteGeometryAvailable.class })
```

Comment:
As far as I understand, testing for the presence of `hibernate-spatial` would be enough? If not, can you please add a comment here explaining why?

Reply:
Geolatte is one of 2 library supported by Spatial for geometric types, the other (and the default) being JTS. I thought the dependency was optional, but you're right - no need to check both.